Azure/sonic-buildimage
# Change dhcp6relay to be a submodule, and rename it to dhcprelay

Signed-off-by: Saikrishna Arcot <sarcot@microsoft.com>



#### Why I did it

`src/dhcprelay` is being split out to be its own submodule.

#### How I did it

1. Add existing dhcprelay commits into the new repo.
2. Clean up Makefile (sonic-net/sonic-dhcp-relay@772625fefcf394e091579288209f4016915f1f1a)
3. Add LGTM config (sonic-net/sonic-dhcp-relay@5cc0889f5c0a0c17f2e4e479ead3e3f81a9be453)
4. Add Azure pipeline config (sonic-net/sonic-dhcp-relay@c79cdb7567a37929ffb1a59cfe3feb5bf1feabb8)
5. Add submodule reference, renaming most references of dhcp6relay to dhcprelay (to reflect that this will not just be for IPv6 in the future).

#### How to verify it

Successful run of LGTM is tested at sonic-net/sonic-dhcp-relay#4. Failure run of LGTM is tested at sonic-net/sonic-dhcp-relay#3.

Azure pipeline is run for each commit/PR, and will build for amd64, armhf, and arm64. UT/code coverage check is not yet done.  

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
